InnerEye/ML/scalar_config.py

```diff
@@ -112,6 +115,11 @@ class ScalarModelBase(ModelConfigBase):
                                             "For binary classification, this field must be a list of size 1, and "
                                             "is by default ['Default'], but can optionally be set to a more descriptive "
                                             "name for the positive class.")
+    target_names: List[str] = param.List(class_=str,
+                                         default=None,
+                                         bounds=(1, None),
+                                         doc="The label names for each output target, used for reporting results. "
```

Comment:
Docstring for class_names conflicts with this docstring. The docstring should also mention that these are the names used for logging, reporting metrics, etc and that target_names takes precedence over class_names if defined.